R/fragments-label.R

```diff
@@ -118,7 +124,8 @@ labelFragments <- function(x, tolerance = 0, ppm = 20, ...) {
 ##' 
 ##' @details
 ##' `addFragments` is deprecated and will be made defunct; use `labelFragments` instead.
-addFragments <- function(x, tolerance = 0, ppm = 20, ...) {
+addFragments <- function(x, tolerance = 0, ppm = 20,
+                         what = c("ion", "mz"), ...) {
```

Comment:
Not sure why you added `what` to this deprecated function? I don't think it should have been added.